tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+add_executable(sof-fuzzer
+	main.c
+	qemu-bridge.c
+	../testbench/topology.c
+	platform/byt-host.c
+)
+
+target_link_libraries(sof-fuzzer PRIVATE
+	"-lrt"
```

Comment:
@ranj063 I think you can use generic logic for this

find_library(LIBRT rt)
if (LIBRT)
    target_link_libraries(sof-fuzzer PRIVATE ${LIBRT})

... otherwise won't work on platforms that don't have librt (android, cygwin).